Resources/Prototypes/Loadouts/role_loadouts.yml

```
@@ -47,396 +66,395 @@
   - ClownShoes
   - Glasses
-  - SurvivalClown
+  - ClownPDADelta # DeltaV
+  - Accessibility # starcup
   - Trinkets
   - ClownJobTrinkets
-  - Accessibility # starcup
+  - SurvivalClown
+  - GroupTankHarness
 
 - type: roleLoadout
   id: JobMime
   canCustomizeName: true
   groups:
-  - GroupTankHarness
   - MimeHead
   - MimeMask
-  - MimeNeck # DeltaV
+  - Scarfs # starcup
   - MimeJumpsuit
+  - MimeBelt
   - MimeBackpack
   - MimeOuterClothing
-  - MimeBelt
   - MimeShoes # starcup
   - Glasses
-  - SurvivalMime
+  - Accessibility # starcup
   - Trinkets
   - MimeJobTrinkets
-  - Accessibility # starcup
+  - SurvivalMime
+  - GroupTankHarness
 
 - type: roleLoadout
   id: JobMusician
   canCustomizeName: true
   groups:
-  - GroupTankHarness
+  - Scarfs # starcup
   - MusicianJumpsuit
   - CommonBackpack
   - MusicianOuterClothing
   - MusicianShoes # starcup
   - MusicianEyewear # starcup
-  - Survival
+  - Accessibility # starcup
   - Trinkets
   - MusicianJobTrinkets
-  - Accessibility # starcup
   - Instruments
+  - Survival
   - GroupSpeciesBreathTool
+  - GroupTankHarness
 
 # Cargo
 - type: roleLoadout
   id: JobQuartermaster
   groups:
-  - GroupTankHarness
   - QuartermasterHead
   - QuartermasterNeck
   - QuartermasterJumpsuit
   - CargoTechnicianBackpack
   - QuartermasterOuterClothing
   - QuartermasterShoes
   - Glasses
-  - Survival
+  - Accessibility # starcup
   - Trinkets
   - QuartermasterJobTrinkets
-  - Accessibility # starcup
+  - Survival
   - GroupSpeciesBreathTool
+  - GroupTankHarness
 
 - type: roleLoadout
   id: JobCargoTechnician
   groups:
-  - GroupTankHarness
   - CargoTechnicianHead
-  - CargoTechnicianNeck # DeltaV
+  - CargoTechnicianNeck # starcup
   - CargoTechnicianJumpsuit
   - CargoTechnicianBackpack
   - CargoTechnicianOuterClothing
-  - CargoPDADelta # DeltaV
   - CargoTechnicianShoes
-  - CargoTechnicianID
   - Glasses
-  - Survival
+  - CargoTechnicianID
+  - Accessibility # starcup
   - Trinkets
   - CargoTechnicianJobTrinkets
-  - Accessibility # starcup
+  - Survival
   - GroupSpeciesBreathTool
+  - GroupTankHarness
 
 - type: roleLoadout
   id: JobSalvageSpecialist
   groups:
-  - GroupTankHarness
-  - SalvageSpecialistNeck # DeltaV
+  - SalvageSpecialistNeck # starcup
   - SalvageSpecialistBackpack
   - SalvageSpecialistOuterClothing
-  - SalvagePDADelta # DeltaV
   - SalvageSpecialistShoes
   - Glasses
-  - Survival
+  - SalvagePDADelta # DeltaV
+  - Accessibility # starcup
   - Trinkets
   - SalvageSpecialistJobTrinkets
-  - Accessibility # starcup
+  - Survival
   - GroupSpeciesBreathTool
+  - GroupTankHarness
 
 # Engineering
 - type: roleLoadout
   id: JobChiefEngineer
   groups:
-  - GroupTankHarness
   - ChiefEngineerHead
+  - ChiefEngineerNeck
   - ChiefEngineerJumpsuit
   - StationEngineerBackpack
-  - ChiefEngineerNeck
   - ChiefEngineerOuterClothing
   - ChiefEngineerShoes
-  - SurvivalExtended
   - Trinkets
   - ChiefEngineerJobTrinkets
   - Accessibility # starcup
```

Order does not match standard noted above
```suggestion
  - Accessibility # starcup
  - Trinkets
  - ChiefEngineerJobTrinkets
```